src/components/form/form.types.ts

```diff
@@ -8,4 +8,6 @@ export interface FormProps<T extends FieldValues> extends AntdFormProps {
     onSubmit: SubmitHandler<T>;
     children: ReactNode;
     defaultValues?: DefaultValues<T>;
+    resetAfterSubmit?: boolean;
+    updateDefaultsAfterSubmit?: boolean;
```

Comment:
can we have cases when we set  updateDefaultsAfterSubmit to false?

Reply:
At this point - no, but it was there just as an option for some weird edge cases that could pop up in the future

Anyway, removed for now.